examples/tlc5947_chain.py

```diff
@@ -0,0 +1,110 @@
+# Simple demo of controlling a chain of several TLC5947 12-bit 24-channel PWM controllers.
+# Will update channel values to different PWM duty cycles.
+# Authors: Tony DiCola, Walter Haschka
+
+import board
+import busio
+import digitalio
+
+import adafruit_tlc5947
+
+# Define pins connected to the TLC5947
+SCK = board.SCK
+MOSI = board.MOSI
```

Comment:
What do you see as advantage of the suggested change?

Reply:
Just more inline with other SPI examples. At least more current ones. The variables aren't used anywhere else in the code, so no need to create them.

Reply:
Ah, I see. Thanks. Though a good compiler would find this and inline by itself, it will most probably ease understanding. I will prepare that.
Do you think that the expression for LATCH just below is to big to do the same? Basically LATCH is also used only once, it only occurs in code commented-out.

Reply:
PR updated; changes done as suggested.